### PR TITLE
Add minify-cssinjs-loader & htmls-webpack-plugin

### PR DESCRIPTION
**Pull request by zaaack**
Add minify-cssinjs-loader in styles loader list, thanks.

### CODE REVIEW

**Comment by alexander-akait** [comment]
Need rebase

**Comment by zaaack** [comment]
@evilebottnawi Done, also added htmls-webpack-plugin, please review again, thanks!

File: README.md [hunk 1]
```diff
@@ -28,9 +31,9 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
     - [Language & Framework](#language--framework)
     - [Utility](#utility)
     - [Testing](#testing)
-  - [Integrations](#integration-libraries)
-  - [Plugins](#webpack-plugins)
-  - [Tools](#webpack-tools)
+  - [Integration Libraries](#integration-libraries)
+  - [Webpack Plugins](#webpack-plugins)
+  - [Webpack Tools](#webpack-tools)
```
Comment (alexander-akait) [inline README.md]: Don't touch this

File: README.md [hunk 2]
```diff
@@ -126,8 +129,9 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list. Feel fr
 - [Less Loader](https://github.com/webpack/less-loader): Less loader module for Webpack. -- *Maintainer*: `Webpack Team` [![Github][githubicon]](https://github.com/webpack)
 - [Stylus Loader](https://github.com/shama/stylus-loader): A stylus loader for webpack. -- *Maintainer*: `Kyle Robinson Young` [![Github][githubicon]](https://github.com/shama) [![Twitter][twittericon]](https://twitter.com/shamakry)
 - [Isomorphic Style Loader](https://github.com/kriasoft/isomorphic-style-loader): Isomorphic CSS style loader for Webpack. -- *Maintainer*: `Kriasoft Team` [![Github][githubicon]](https://github.com/kriasoft) [![Twitter][twittericon]](https://twitter.com/kriasoft)
-- [Mini css extract plugin](https://github.com/webpack-contrib/mini-css-extract-plugin): 
+- [Mini css extract plugin](https://github.com/webpack-contrib/mini-css-extract-plugin):
```
Comment (alexander-akait) [inline README.md]: Here loader list, not plugin

File: README.md [hunk 3]
```diff
@@ -211,6 +215,8 @@ Lightweight CSS extraction plugin -- *Maintainer*: `Webpack Contrib` [![Github][
 - [Public Path Manipulation Plugin](https://github.com/agoldis/webpack-require-from) - control `publicPath` of dynamically loaded resources at runtime  -- *Maintainer*: `Andrew Goldis` [![Github][githubicon]](https://github.com/agoldis)
 - [Build Notifier Plugin](https://github.com/roccoc/webpack-build-notifier) - Display OS-level notifications for build errors and warnings. -- *Maintainer*: `Rocco Cataldo` [![Github][githubicon]](https://github.com/roccoc)
 - [Chrome Extension Reloader](https://github.com/rubenspgcavalcante/webpack-chrome-extension-reloader) - Hot reloading while developing Chrome extensions -- *Maintainer*: `Rubens P. G. Cavalcante` [![Github][githubicon]](https://github.com/rubenspgcavalcante) [![Twitter][twittericon]](https://twitter.com/rubenspgc)
+- [Htmls Webpack Plugin](https://github.com/zaaack/htmls-webpack-plugin): Simple and fast multiple-htmls-generating plugin for webpack. -- *Maintainer*: `Zack Young` [![Github][githubicon]](https://github.com/zaaack) [![Twitter][twittericon]](https://twitter.com/ZaaackYoung)
+
```
Comment (alexander-akait) [inline README.md:216]: Need tests for plugin and more description

**Review by alexander-akait — CHANGES_REQUESTED** [review]
Some notes

File: README.md (hunk 1 above)
Comment (zaaack) [reply README.md]: Sorry, vscode updates readme's TOC automatically, I'll edit it via sublime.

File: README.md (hunk 2 above)
Comment (zaaack) [reply README.md]: Actually, this is not my plugin, and it's already there...But I can move it to plugins.

File: README.md (hunk 3 above)
Comment (zaaack) [reply README.md:216]: Done.

File: README.md (hunk 2 above)
Comment (alexander-akait) [reply README.md]: @zaaack thanks :+1:

File: README.md (hunk 1 above)
Comment (alexander-akait) [reply README.md]: :+1:

File: README.md (hunk 3 above)
Comment (alexander-akait) [reply README.md:216]: :+1:

**Comment by zaaack** [comment]
@evilebottnawi Can this PR been merged now?

**Comment by zaaack** [comment]
@evilebottnawi  Thanks!

**Pull request merged**